### PR TITLE
Enhancement/cookbook scroll up

### LINKED ISSUE #669: [Enhancement] Scroll up when selecting a tab/menupoint
We want the site to scroll up when we select a tab on the left side. Right now it stays where you are located.

### PR DESCRIPTION
**Pull request by MadsBuchmann**
This PR closes #669

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [X] cookbook application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: #669 

## What is the new behavior?
Whenever a route change happens (i.e. when a menu item is clicked) the scroll position will be restored to the top of the page. 

## Does this PR introduce a breaking change?

- [ ] Yes
- [X] No

### CODE REVIEW

File: apps/cookbook/src/app/app-routing.module.ts [hunk 1]
```diff
@@ -4,7 +4,12 @@ import { RouterModule } from '@angular/router';
 import { routes } from './app.routes';
 
 @NgModule({
-  imports: [RouterModule.forRoot(routes)],
+  imports: [
+    RouterModule.forRoot(routes, {
+      scrollOffset: [0, 0],
```
Comment (jakobe) [inline apps/cookbook/src/app/app-routing.module.ts:9]: Is this needed? 🤔

File: apps/cookbook/src/app/app-routing.module.ts [hunk 2]
```diff
@@ -4,7 +4,12 @@ import { RouterModule } from '@angular/router';
...
+      scrollPositionRestoration: 'enabled',
```
Comment (jakobe) [inline apps/cookbook/src/app/app-routing.module.ts:10]: Nice! 👌 
Didn't know about that one, nice to learn something new 🤓

**Review by jakobe — APPROVED** [review]
Consider reverting the 3 files unrelated to the actual PR/issue...

File: apps/cookbook/src/app/app-routing.module.ts (hunk 1 above)
Comment (MadsBuchmann) [reply apps/cookbook/src/app/app-routing.module.ts:9]: Are you referring to `scrollOffset`?

Comment (jakobe) [reply apps/cookbook/src/app/app-routing.module.ts:9]: Yes

Comment (MadsBuchmann) [reply apps/cookbook/src/app/app-routing.module.ts]: I initially tought it was - but just tested without and it is not. I've removed it in a new commit 😄

**Review by jakobe — APPROVED** [review]

**Pull request merged**